src/search/landmarks/landmark_factory.cc

```diff
@@ -389,6 +397,18 @@ void LandmarkFactory::approximate_reasonable_orders(
     }
 }
 
+bool LandmarkFactory::have_common_achiever(
+    const LandmarkNode *node_a, const LandmarkNode *node_b) const {
+    set<int> achievers_a = node_a->possible_achievers;
```

Comment:
Is it on purpose that these are copied? I think this should be a reference instead.